tcf_website/templates/base/sidebar.html

```diff
@@ -1,35 +1,49 @@
 {% load static %}
 
 <!-- Sidebar -->
-<div class="border-right
-d-flex flex-column justify-content-between" id="sidebar-wrapper">
+<div id="sidebar-wrapper" class="border-right pb-0
+     d-flex flex-column justify-content-between text-center">
   <div class="list-group list-group-flush">
     <!-- TODO: dashboard for saved courses, enrollment dates, major-specific info, etc -->
```

Comment:
I'd take out this comment, there are other places where we can store TODO items

Reply:
done